src/Massive.Shared.Async.cs

```diff
@@ -127,7 +127,6 @@ public Task<List<dynamic>> QueryAsync(string sql, DbConnection connection, param
 				{
 					toReturn.Add(rdr.RecordToExpando());
 				}
-				rdr.Close();
```

Comment:
This can't be removed on .NET full. reader.Close() is an optimization for SQL Server, as it otherwise is considerably slower (why, no-one knows)